### PR TITLE
OD-201 [Feature] Update security rules

### PR DESCRIPTION
**Pull request by AndrRyaz**
@sofiiakvasnevska 
## Issue
OD-201 https://weboo.atlassian.net/jira/software/projects/OD/boards/9?selectedIssue=OD-201
## Description
Update security rules
## Screenshots/screencasts
https://screencast-o-matic.com/watch/cYXY6ULTJW
## Backward compatibility
This change is fully backward compatible.
## Reviewers
@upplabs-alex-levchenko

### CODE REVIEW

**Comment by AndrRyaz** [comment]
@tonytlwu In the process of solving the problem, I came across a little confusion, should the radio button have an array?
I think this creates a lot of unnecessary work
https://github.com/Fliplet/fliplet-widget-dynamic-lists/pull/447/files#diff-3e00189c2b38ee2b81964659c3a01eceef437f47aeefe02a20fa6f8d876a938fR467

**Comment by tonytlwu** [comment]
@AndrRyaz [The link you provided](https://github.com/Fliplet/fliplet-widget-dynamic-lists/pull/447/files#diff-3e00189c2b38ee2b81964659c3a01eceef437f47aeefe02a20fa6f8d876a938fR467) doesn't seem to provide any context for what you're talking about.

Can you be more specific on what the problem is and outline what you suggest the alternative could be?

**Comment by AndrRyaz** [comment]
@tonytlwu 
image

this screen shows that editRadioValue is an array, although this variable is responsible for the radio button, but the radio button should be responsible for one value, it cannot have an array, as I understand it

**Comment by tonytlwu** [comment]
@AndrRyaz That's correct. I can see that this has always been a radio input from the first commit of this feature. https://github.com/Fliplet/fliplet-widget-dynamic-lists/pull/41

Is this creating any problem in your work of updating security rules?

**Comment by AndrRyaz** [comment]
@tonytlwu no, solution are working but i note this because it not optimized

**Comment by tonytlwu** [comment]
@AndrRyaz Optimisation is out of scope

**Comment by AndrRyaz** [comment]
@tonytlwu ok, so can you please review the pr and say if its good or not.

File: js/interface.js [hunk 1]
```diff
@@ -687,6 +706,15 @@ var DynamicLists = (function() {
         }
       });
     },
+    toggleRuleType: function(type, isTypeActive) {
+      var typeIndex = accessRules[0].type.indexOf(type);
+
+      if (isTypeActive && typeIndex === -1) {
+        accessRules[0].type.push(type);
```
Comment (tonytlwu) [inline js/interface.js:713]: See https://github.com/Fliplet/fliplet-widget-form-builder/pull/350/files#r531022558

**Comment by tonytlwu** [comment]
See https://github.com/Fliplet/fliplet-widget-form-builder/pull/350#issuecomment-734256547

**Review by tonytlwu — APPROVED** [review]
@sofiiakvasnevska This is ready for testing

Risk: High
Affected: Studio users configuring LFD data sources
Verify: Users can configure LFD settings

Squash and merge: Update security rules

**Pull request merged**